File: ZhetistikApp.Api/Models/Candidate.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ZhetistikApp.Api.Models
{
    [Table("Candidates")]
    public class Candidate
    {
        [Key]
        public int CandidateID { get; set; }
        public int UserID { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public DateTime Birthday { get; set; }
        public string Email { get; set; }
        public string PhoneNumber { get; set; }

        public Candidate()
        {

        }

        public Candidate(int candidateID, int userID, string firstName, string lastName, DateTime birthday, string email, string phoneNumber)
        {
            CandidateID = candidateID;
            UserID = userID;
            FirstName = firstName;
            LastName = lastName;
            Birthday = birthday;
            Email = email;
            PhoneNumber = phoneNumber;
        }
    }
}

File: zhetistik.ui/src/Candidate.js
import React, {Component} from 'react';
import {Table} from 'react-bootstrap'

export class Candidate extends Component
{
    constructor(props)
    {
        super(props);
        this.state={candidates:[]}
    }
    refreshList()
    {
        fetch(process.env.REACT_APP_API+'candidates/')
        .then((response) => {
            return response.json();
          })
        .then(data=>{
            this.setState({candidates:data});
        });
    }
    componentDidMount(){
        this.refreshList();
    }
    componentDidUpdate(){
        this.refreshList();
    }
    render(){
        const {candidates} = this.state;
        return(
            <div>
                <Table className="mt-4" striped bordered hover size='sm'>
                    <thead>
                        <tr>
                        <td>CandidateID</td>
                        <td>FirstName</td>
                        <td>LastName</td>
                        <td>Birthday</td>
                        <td>Email</td>
                        <td>PhoneNumber</td>
                        </tr>
                    </thead>
                    <tbody>
                        {candidates.map(can=>
                            <tr key={can.CandidateID}>
                                <td>{can.CandidateID}</td>
                                <td>{can.FirstName}</td>
                                <td>{can.LastName}</td>
                                <td>{can.Birthday}</td>
                                <td>{can.Email}</td>
                                <td>{can.PhoneNumber}</td>
                                <td>Edit / Delete</td>
                            </tr>
                        )}
                    </tbody>
                </Table>
            </div>
        )
    }
}